definitions/contested/json/CaseEventToFields/CaseEventToFields.json

```diff
@@ -10071,6 +10071,7 @@
     "PageID": 3,
     "PageDisplayOrder": 3,
     "PageColumnNumber": 1,
+    "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/case-orchestration/ccdAboutToSubmitEvent",
```

Comment:
I don't think this is a good idea to use the about to submit handler fro the mid-event

Reply:
I had a go at using the MidHandler. I found the validation wouldn't work correctly when the user removes the inputted data in the postcode field. The MidHandler would store the previous inputted information even when the user removes which prevents the validation from working appropriately. I can try this again to confirm.